app/components/mandatarissen/fractie-selector.js

```diff
@@ -29,18 +29,64 @@ export default class MandatenbeheerFractieSelectorComponent extends Component {
   }
 
   async loadFracties() {
-    let fracties = await this.fetchFracties();
+    let fracties = [];
+
+    if (this.isUpdatingFractie && this._fractie) {
```

Comment:
Do you need a dedicated function for this? Can't you just use this.args.isUpdatingFractie? Also might be better to use another variable name, since isUpdatingFractie is a bit vague, in the other use case, you also update the fractie ... maybe better something like "isUpdatingState"?

Reply:
Updated the args value that's indeed some code thats not needed there. 👌🏻 
Hm I thought about to but when checking the actions and button names I came out on `isUpdatingFractie` becasue the other action is `correcting` the fractie. Just using the `isUpdatingState` name will not be enough because the component is always updating the fractie ;p

Let's see on monday what the naming says :D